azure-iot-provisioning-devicesdk/samples/register_symmetric_key.py

```diff
@@ -0,0 +1,52 @@
+import os
+import logging
+from azure.iot.provisioning.devicessdk.security.sk_security_client import SymmetricKeySecurityClient
```

Comment:
>security.sk_security_client [](start = 39, length = 27)

User should not be having to import from specific submodules and subpackages. Expose this all through the devicesdk `__init__.py`